docs/sources/api/_index.md

```diff
@@ -1044,7 +1048,13 @@ curl -g -X POST \
   -H 'X-Scope-OrgID: 1'
 ```
 
-The query parameter can also include filter operations. For example `query={foo="bar"} |= "other"` will filter out lines that contain the string "other" for the streams matching the stream selector `{foo="bar"}`.
+The same example deletion request for Grafana Enterprise Logs uses Basic Authentication and specifies the tenant name as a user; `Tenant1` is the tenant name in this example. The password in this example is an access policy token that has been defined in the API_TOKEN environment variable:
```

Comment:
```suggestion
The same example deletion request for Grafana Enterprise Logs uses Basic Authentication and specifies the tenant name as a user; `Tenant1` is the tenant name in this example. The password in this example is an access policy token that has been defined in the API_TOKEN environment variable. The token must be for an access policy with `logs:delete` scope for the tenant specified in the user field:
```

Reply:
It pains me to put this sentence in here because it feels like it belongs elsewhere but since we don't have an elsewhere to put it for now, I think this is our best bet. WDYT @KMiller-Grafana ?

Reply:
We'll put it elsewhere when we have a place to put it.  No worries for this short term solution.